coordinator/coordinator.go

```diff
@@ -340,7 +340,7 @@ func (c *coordinator) saveCheckpointTs(ctx context.Context, cfs map[common.Chang
 			continue
 		}
 		cf.SetLastSavedCheckPointTs(cp)
-		if cf.IsMQSink() {
+		if cf.IsMQSink() || cf.IsStorageSink() {
```

Comment:
IsMQSink and IsStorageSink are only used here. So there is no need to separate these two functions. We can use a unit function (such as `NeedCheckpointTsMessage`) to contain the entire logic

Reply:
IsMQSink and IsStorageSink are also used in the unit tests.

Reply:
The unit test just use to check the changefeed is correctly set the field. We can change a unity function, and do the check.